COMPATIBILITY.md

```diff
@@ -63,7 +63,8 @@ Newly filed issues should bear the label `pipeline` for ease of tracking.
 - [X] `GitHubCommitNotifier`, `GitHubSetCommitStatusBuilder` (`github`): scheduled to be supported as of 1.14.3
 - [ ] `CoverityPublisher` (`coverity`): [JENKINS-32354](https://issues.jenkins-ci.org/browse/JENKINS-32354)
 - [X] `XUnitPublisher` and `XUnitBuilder` (`xunit`): scheduled to be supported as of 1.100
-- [ ] `PerformancePublisher` (`performance`): [JENKINS-32650](https://issues.jenkins-ci.org/browse/JENKINS-32650)
+- [X] `PerformanceTest` (`bzt`,`performanceTest`): supported as of 3.1
+- [X] `PerformancePublisher` (`perfReport`,`performanceReport`): supported as of 3.1
```

Comment:
AFAICT there are no plugins named `bzt`, etc. Only `performance`, as in the original line.